charmbenchmark/__init__.py

```diff
@@ -75,18 +75,30 @@ def set_data(value):
             return False
 
     @staticmethod
-    def set_composite_score(value, units, direction='asc'):
+    def set_composite_score(value, units='', direction=None):
         """
         Set the composite score for a benchmark run. This is a single number
         representative of the benchmark results. This could be the most
         important metric, or an amalgamation of metric scores.
         """
         Benchmark.set_data({'meta.composite.value': value})
```

Comment:
knit pick, but why not use Benchmark.set_meta here? Not a blocker for this PR, but something to consider in the future. It'll simplify testing and future proof changes to meta